x/ref/examples/main.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2015 The Vanadium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// An example using the 'library' factory which is configured via exported
+// variables rather than by command line flags.
+package main
+
+import (
+  "v.io/v23"
+  "v.io/x/ref/runtime/factories/library"
+)
+
+func init() {
```

Comment:
Does this need to in done in an `init()` function?

In general `init()` functions complicates a bit the flow in a program so it's good to only use them when absolutely necessary.